docs/source/torch_xla.rst

```diff
@@ -0,0 +1,92 @@
+torch_xla
```

Comment:
We pervasively use Markdown for documentation.
Can we use that instead of RST?

Reply:
I'll check with our docs team.